libpod/runtime.go

```diff
@@ -19,6 +19,7 @@ import (
 	sysreg "github.com/projectatomic/libpod/pkg/registries"
 	"github.com/sirupsen/logrus"
 	"github.com/ulule/deepcopier"
+	"strconv"
```

Comment:
Gofmt is going to complain about this

Reply:
naw, gofmt doesnt care about that.  but i fixed it